man/systemd-keyutil.xml

```diff
@@ -119,6 +120,15 @@
         <xi:include href="version-info.xml" xpointer="v258"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--hash-alg=<replaceable>ALG</replaceable></option></term>
+
+        <listitem><para>Hash algorithm used to generate the PKCS#1 signature for the <command>pkcs7</command>
+        command. Defaults to SHA-256.</para>
```

Comment:
Looking at the code and test, `Defaults to <literal>sha256</literal>.` or `<literal>SHA256</literal>` should be better?

Or `"SHA-256"` is also acceptable?

Reply:
yep that is better, i updated it to use `<literal>SHA256</literal>` and also added a comment that a list of valid algorithms can be seen by running `<literal>openssl list -digest-algorithms</literal>` (I'm not sure if i should literal there or a different tag?)